packages/editor/src/components/post-author/panel.js

```diff
@@ -16,7 +17,8 @@ import { useAuthorsQuery } from './hook';
 
 function PostAuthorToggle( { isOpen, onClick } ) {
 	const { postAuthor } = useAuthorsQuery();
-	const authorName = postAuthor?.name || '';
+	const authorName =
+		decodeEntities( postAuthor?.name ) || __( '(No author)' );
```

Comment:
Entity decoding is also needed here.

## Screenshot
![CleanShot 2024-08-02 at 12 01 30](https://github.com/user-attachments/assets/5307db5d-45d7-4c28-86d0-cfc69f678578)